Frontend/app/src/androidTest/java/io/happybugs/happybugs/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package io.happybugs.happybugs;
+
+import android.content.Context;
```

Comment:
Group all imports in one group, and all static imports in another group. https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing